src/components/ArticleCard.jsx

```diff
@@ -35,6 +38,30 @@ export default function ArticleCard() {
     );
   }
 
+  function handleUpVote() {
+    setVoted((currVotes) => {
```

Comment:
Your voting logic seems to only optimistically render and never patches to the database. It looks like you've got a bit confused here about how to handle the vote. We need to:

- optimistically render using the state for a newVote here. We can use our setVoted function to add one to the previous state - no need to return an object
- then make a call to the database with patch to update the actual votes

Reply:
Yeah, I realised this as soon as I submitted a PR. 🤦‍♀️ sorry! I've been working on it for the last 45 minutes, but I might need to put a /nchelp :/